webapp/components/form/Input/NumberFormats.js

```diff
@@ -0,0 +1,64 @@
+export const integer = () => ({
+  decimalScale: 0,
+  maxLength: 16,
+})
+
```

Comment:
fileName can be lowerCase since it's not a component

Reply:
right